Dockerfile-alpine.template

```diff
@@ -43,4 +43,20 @@ RUN adduser -D -u 1000 node \
     && rm -Rf "node-v$NODE_VERSION" \
     && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
+ENV YARN_VERSION 0.0.0
+
+RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg \
+  && for key in \
+    6A010C5166006599AA17F08146C2130DFD2497F5 \
+  ; do \
+    gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
+  done \
+  && curl -SL -o yarn.js "https://github.com/yarnpkg/yarn/releases/download/v$YARN_VERSION/yarn-legacy-$YARN_VERSION.js" \
```

Comment:
Please use `https://yarnpkg.com/downloads/0.18.2/yarn-legacy-0.18.2.js` and `yarnpkg.com/downloads/0.18.2/yarn-legacy-0.18.2.js.asc` for the download URLs :)